social_core/backends/weixin.py

```diff
@@ -53,7 +53,8 @@ def auth_params(self, state=None):
         appid, secret = self.get_key_and_secret()
         params = {
             'appid': appid,
-            'redirect_uri': self.get_redirect_uri(state)
+            'redirect_uri': self.get_redirect_uri(state),
+            'scope': 'snsapi_login',
```

Comment:
This should be in the `DEFAULT_SCOPE` attribute in the class, the lib will pick the value and add it to the URL automatically.